sound/soc/sof/sof-pci-dev.c

```diff
@@ -10,6 +10,7 @@
 
 #include <linux/firmware.h>
```

Comment:
do we need to refine following header files for intel? 

#ifdef CONFIG_X86

 /* platform specific devices */
 #include "intel/shim.h"
#include "intel/hda.h"

#endif